index.js

```diff
@@ -2,13 +2,17 @@ const express = require('express');
 const app = express();
 app.use(express.static('public'))
 app.set('view engine', 'ejs');
+const date = require('date-and-time')
+const now  =  new Date();
+app.locals.serverStartTime = date.format(now,'YYYY/MM/DD HH:mm:ss');
 
 app.get('/', (req, res) => {
     res.render('index', {
         title: process.env.TITLE || 'The system is currently undergoing maintainance',
         message: process.env.MESSAGE || 'We are currently undergoing maintainance. Please check back later.',
         supportPhone: process.env.SUPPORT_PHONE || '+00 000 000 0000',
-        supportEmail: process.env.SUPPORT_EMAIL || 'mail@address.com'
+        supportEmail: process.env.SUPPORT_EMAIL || 'mail@address.com',
+        serverStartTime: app.locals.serverStartTime
```

Comment:
The default value for the phone number should be a dash or something.
The default value for the support email should be info@ozone-his.com.

@rbuisson what do you think of the latter?

Reply:
I would assume something more like support@ozone-his.com
But otherwise, in regards to defaulting to an ozone-his.com email address, I'm  fine with this. That means that if a user of FOSS does not override this value, emails would be sent to us instead. That works.

Reply:
My issue with support@ozone-his.com is that it gives the impression that they are entitled to support, which might not be the case at all in the case of an implementation made "out there".

Reply:
Correct, so I was thinking for those who don't have support, we answer to the email and explain the support model and redirect to the purchase of a pro plan.
Or, we have a default message on FOSS that tells the same thing.